config.yaml

```diff
@@ -17,21 +27,16 @@ common:
    --plugin-option=tmpfs:keep_mounted=True
    --plugin-option=tmpfs:max_fs_size=32g
    --plugin-option=tmpfs:required_ram_mb=39800"
- commit_updates: True
- push_updates: True
-build_packages:
- packages:
- packages_repos_target_path: "/var/lib/host-os/repositories"
- keep_build_dir: False
- result_dir:  "./result"
- force_rebuild: False
 build_release_notes:
  repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  repo_branch: "master"
+ push_repo_url: "git@github.com:open-power-host-os/open-power-host-os.github.io.git"
```

Comment:
I do not understand why we should not push directly to the main repository. In the case of our team, that's exactly what we want to do. You are afraid that a member of our team will run "host_os.py --build-release-notes" accidentally? If that is a possibility, is forcing him to use "host_os.py --build-release-notes --push-repo-url="git@github.com:open-power-host-os/open-power-host-os.github.io.git"  safer?

In the case of someone outside our team, he/she will not have permission to do so, as you mentioned, so there is no issue IMHO.